sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -234,10 +235,11 @@ object UnresolvedAttribute {
 case class UnresolvedGenerator(name: FunctionIdentifier, children: Seq[Expression])
   extends Generator {
 
-  override def elementSchema: StructType = throw new UnresolvedException("elementTypes")
-  override def dataType: DataType = throw new UnresolvedException("dataType")
-  override def foldable: Boolean = throw new UnresolvedException("foldable")
-  override def nullable: Boolean = throw new UnresolvedException("nullable")
```

Comment:
I think these are assert-like errors that end-users should not hit. Can we keep them here?

Reply:
OK